File: cfgs/java_cfgs_foo/FIND_THE_MISSING_NUMBER_1.java
Nodes
1;special;;;
2;statement;int total = 1;14;
3;statement;int i = 2;15;
4;conditional;foo_1 <= (foo_2 + 1);16;
5;statement;total += foo_1;18;
6;statement;total -= foo_1[foo_2 - 2];19;
7;statement;i++;17;
8;statement;return foo_1;21;
9;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;8;
4;6;
5;7;
6;4;
7;9;
8;

File: cfgs/pythoncfgs/FIND_THE_MISSING_NUMBER_1.py
Nodes
1;statement;(foo_1, foo_2) = (0, 1);8;
2;conditional;foo < foo;9;
3;statement;return foo_1;12;
4;statement;foo_1 += foo_2;10;
5;statement;foo_1 -= foo_2[foo_3 - 2];11;
6;exit;;
Edges
0;2;
1;3;4;
2;6;
3;5;
4;2;
5;
